cases/v41/const.py

```diff
@@ -41,91 +28,53 @@
 ]
 dashboard_nodectl_id = [18545, 18547, 18550, 18551, 18552, 18830, 18831]
 
-
-# test_dashboard_ui
 dashboard_ui = [
     'check_node_status_func', 'check_node_health_func', 'check_node_info_func',
     'check_network_func', 'check_system_log_func', 'check_storage_func',
-    'check_ssh_key_func'
-]
-dashboard_ui_id = [18534, 18535, 18536, 18540, 18541, 18542, 18543]
-
-
-# test_dashboard_ui_efi
-dashboard_ui_efi = [
-    'check_nodestatus_efi'
+    'check_ssh_key_func', 'check_nodestatus_efi', 'check_nodestatus_fc'
 ]
-dashboard_ui_efi_id = [18539]
+dashboard_ui_id = [18534, 18535, 18536, 18540, 18541, 18542, 18543, 18539, 18538]
 
 
-# test_dashboard_ui_fc
-dashboard_ui_fc = [
-    'check_nodestatus_fc'
+he_install = [
+    'check_he_install', 'check_he_install_bond', 
+    'check_he_install_vlan', 'check_he_install_bv'
```

Comment:
check_he_install_bond, check_he_install_vlan, check_he_install_bv are target for installing HE over a bond/vlan/bond+vlan network device, so they are mutual exclusion in each test. Please just keep them as original.

Reply:
Not sure about these changes as well